swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
+from curses import wrapper
+from re import T
+from signal import set_wakeup_fd
+from pytest import Item
+from swap_meet.item import Item 
+
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory =  None):
+        if inventory == None:
+            inventory = list(())
+        self.item = Item()
+        self.inventory = inventory
+
+            
+    def add(self, item):
+        self.item = item 
```

Comment:
you can remove this line. The item being passed in as a parameter is what will be used in `self.inventory.append(item)`